examples/democoin/app/app.go

```diff
@@ -65,12 +65,15 @@ func NewDemocoinApp(logger log.Logger, db dbm.DB) *DemocoinApp {
 		&types.AppAccount{}, // prototype
 	).Seal()
 
+	// Generate codespacer
+	codespacer := sdk.NewCodespacer()
+
 	// Add handlers.
 	coinKeeper := bank.NewCoinKeeper(app.accountMapper)
-	coolKeeper := cool.NewKeeper(app.capKeyMainStore, coinKeeper)
-	powKeeper := pow.NewKeeper(app.capKeyPowStore, pow.NewPowConfig("pow", int64(1)), coinKeeper)
-	ibcMapper := ibc.NewIBCMapper(app.cdc, app.capKeyIBCStore)
-	stakeKeeper := simplestake.NewKeeper(app.capKeyStakingStore, coinKeeper)
+	coolKeeper := cool.NewKeeper(app.capKeyMainStore, coinKeeper, codespacer.Register(cool.DefaultCodespace))
```

Comment:
.. so could be app.RegisterCodespace(cool.DefaultCodespace)

Reply:
Updated